### PR TITLE
Build project

### PR DESCRIPTION
**Pull request by prolajumokeoni**
- Build a clone of the original New York Times article page.
-  Embed videos using HTML 5 tag.
- Position elements using Flexbox, Float, and Position Absolute/Relative/Fixed.
-  Deploy on GitHub pages.

> I have effected all the changes requested by the previous TSE reviewer.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+        <div class="search-icon">
+          <i class="fas fa-search"></i>
+        </div>
+        <div class="space">
+          <a class="space-cosmos" href="#">SPACE & COSMOS</a>
+        </div>
+      </div>
+      <div class="header-logo">
+        <img  src="./assets/images/svgexport-3.svg" alt="">
+      </div>
```
Comment (teekaytech) [inline index.html]: - [x] The NY Times logo should be a link (Use `<a>` tag around the image).

File: index.html [hunk 2]
```diff
@@ -0,0 +1,570 @@
...
+                <li class="edu">Education Rate</li>
+                <li class="listings">Mobile Applications</li>
+                <li class="listings">Replica Edition</li>
+                <li class="listings">International</li>
+                <li class="listings">Cananda</li>
+                <li class="listings">Espanol</li>
+                <li class="listings">中文网</li>
+              </ul>
+          </li>  
+        </ul>
```
Comment (teekaytech) [inline index.html]: - [x] You should use an anchor tag with any valid value e.g. "#" for the href attribute.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,570 @@
...
+          <li class="footer-items"><a class="footer-list" href="">T Brand Studio</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Your Ad Choices</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Privacy Policy</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Terms of Service</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Terms of Sale</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Site Map</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Canada</a></li>
+          <li class="footer-items"><a class="footer-list" href="">International</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Help</a></li>
+          <li class="footer-items"><a class="footer-list" href="">Subscriptions</a></li>
```
Comment (teekaytech) [inline index.html]: - [x] Kindly use a valid value e.g. "#"  for the href attributes

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @prolajumokeoni  :wave:

__You've really done well so far. Here're the major highlights of this PR:__

- Your project has a descriptive README :+1: 
- Your Pull Request has a meaningful title and a brief description :ok_hand: 
- All linter checks are running successfully :100: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] Files not related to the project should not be committed and added to .gitignore (e.g. .vscode). Since you've already committed the `vscode` directory, kindly delete it locally, add the directory name to `.gitignore`, then commit your changes.
- [x] Please go through your HTML file and ensure that all `alt` attributes of the images tag are given their corresponding values. No `alt` attribute should have a blank (" ") value.
- [x] The image tags and headers in the `More in Space and Astronomy` section should be linked.
- [x] Also, you should use anchor tags `<a>` for items in the `Most Popular` section. Do not forget to use the `#` symbol for the href attributes.
- [x] Kindly check my inline comments below and make necessary changes.

**[ OPTIONAL CHANGES ]**

- I strongly suggest you reduce the size of the font in the `Theory of Inflation` section. This will make the text fit better into the spaces of the background image.
- It will be nice if you can add the fixed footer (the footer that sticks to the bottom part of the screen, as we have on the original page) 

* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comment and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 code reviews left for this project.**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

File: index.html (hunk 3 above)
Comment (prolajumokeoni) [reply index.html]: > * [x]  Kindly use a valid value e.g. "#"  for the href attributes

Thank you for your review, I have implemented all the changes, I will submit for a re-review.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hi @prolajumokeoni :wave:

__Thank you for implementing most of the changes required by the last reviewer. I must also acknowledge your effort in implementing the optional changes. You've really done well so far :handshake: Your webpage is looking clean and we're almost good to go!__

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [x] You're still having the `.vscode` directory in your remote folder structure. Let's fix this. 
The easiest way to do this is to:
- Delete the `.vscode` locally from your project root directory
- Add the `.vscode` directory to your `.gitignore` file. I can see you did this already, however, you probably skip the initial `.`, which is part of the directory name (i.e. `/.vscode`).
- Add and commit your changes.
- You can then restore the `.vscode` directory from the Cycle bin to your project if you still need it for developmental purposes.
- [x] The NY Times logo in the fixed top navbar should be a link (Use <a> tag around the image).



* * *
**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

**REMINDER: As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 1 code review left for this project. If you think that the code review was not fair, you can [request a second-opinion using this form](https://airtable.com/shrmBIv6cXYrq4Fgv).**

_**Please make the required changes and submit another review request when you are done.**_

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by prolajumokeoni** [comment]
Thank you for the re-review I have effected all the changes requested @teekaytech. Can you help me check to see if I did the changes correctly before I submit?

**Review by teekaytech — APPROVED** [review]
## Status: Approved :green_circle: 

### Congratulations @prolajumokeoni :tada: :clinking_glasses: 

_**Thank you for fixing all the changes required earlier.**_

**This project has passed all the mandatory requirements :clap: 
It is therefore approved :heavy_check_mark:**

Again, congratulations, you can merge to the default branch.
Do not forget to fill the project completion form, as this is important for your weekly progress score.

**_Good luck with your next task._**

* * * 
**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Comment by prolajumokeoni** [comment]
> ## Status: Approved 🟢
> ### Congratulations @prolajumokeoni 🎉 🥂
> _**Thank you for fixing all the changes required earlier.**_
> 
> **This project has passed all the mandatory requirements 👏 It is therefore approved ✔️**
> 
> Again, congratulations, you can merge to the default branch.
> Do not forget to fill the project completion form, as this is important for your weekly progress score.
> 
> **_Good luck with your next task._**
> 
> **Taofeek Olalere [Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
> **Happy Coding 😃**

Thank you for the reviews @teekaytech

**Comment by prolajumokeoni** [comment]
Special thanks to my mentors  @ezeilo-su & @ShoiraTa  for their guidance.

**Comment by prolajumokeoni** [comment]
Special thanks to my mentors @ezeilo-su & @ShoiraTa for their guidance.

**Pull request closed without merging**